tests/phpunit/CRM/Core/Payment/BaseIPNTest.php

```diff
@@ -411,82 +411,63 @@ public function testRequiredWithContributionPageError() {
     $this->assertFalse(is_array($result));
   }
 
-  /* @codingStandardsIgnoreStart
-     * Test calls main functions in sequence per 'main' - I had hoped to test the functions more
-     * fully but the calls to the POST happen in more than one function
-     * keeping this as good example of data to bring back to life later
-
-    public function testMainFunctionActions() {
```

Comment:
is nothing lost by this removal?

Reply:
nope, just the commented out code is removed. but you are welcome to double check :)